### DISCUSSION TITLE
Make profiles compatible with Firefox

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#849] [Ideas]
### What feature would you like to see?

As it stands, attempting to copy a Firefox profile over to Zen doesn't work (Even if both are on the same version), and Zen refuses to open claiming the profile is much newer than the browser.

This happens as Firefox saves its version in the profile's `compatibility.ini` (In my case 129) and Zen compares it to its own version (In my case, 1.0.0-a26).

Fixing this or allowing an auto-import which would replace the version could make transitioning from Firefox to Zen much easier.

(After patching `compatibility.ini` to the correct Zen version, the profile loaded without any issue)

### What platform are you seeing the problem on?

Windows

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/401_

**Comment by SomeDudeBrosky** [comment]
Just log in using the header bar.

**Reply by thibaultmol** [reply]
What do you mean?
